test/integration/ComputeExamples.test.ts

```diff
@@ -661,6 +662,8 @@ describe('Compute-to-data example tests', async () => {
     /// Let's save the compute job it, we re going to use later
     /// ```Typescript
     computeJobId = computeJobs[0].jobId
+    // eslint-disable-next-line prefer-destructuring
```

Comment:
This should be written with three slashes in this file `///` - the ComputeExamples.md file is auto-generated from this file and one of the slashes gets removed in the process. 

```suggestion
    /// eslint-disable-next-line prefer-destructuring
```